adhocracy4/static/__tests__/TermsOfUseCheckbox.jest.jsx

```diff
@@ -1,10 +1,11 @@
 import React from 'react'
 import { render, screen, fireEvent } from '@testing-library/react'
 import { TermsOfUseCheckbox } from '../TermsOfUseCheckbox'
+import django from '../../../../../__mocks__/djangoMock'
 
 test('One TermsOfUseCheckbox is showing', () => {
   render(<TermsOfUseCheckbox id="test-checkbox" />)
-  const component = screen.getByLabelText(/mock text/)
+  const component = screen.getByLabelText(django.interpolate(), /mock text/)
```

Comment:
because the test is getting the label text and we now use interpolate in it, it needed a django mock to be able to run the test. Is that right @khamui @goapunk ? i'm still bit confused by them myself?!